components/Buttons/examples/ButtonsShapesExampleViewController.m

```diff
@@ -14,8 +14,8 @@
  limitations under the License.
  */
 
-#import "MDCButtonColorThemer.h"
-#import "MDCButtonTypographyThemer.h"
+#import "MaterialButtons+ColorThemer.h"
```

Comment:
`#import "MaterialButtons+ButtonThemer.h"` instead of both?

Reply:
Done.